ivy/data_classes/container/experimental/creation.py

```diff
@@ -862,3 +862,98 @@ def unsorted_segment_min(
             segment_ids,
             num_segments,
         )
+
+
+@staticmethod
+    def static_unsorted_segment_max(
```

Comment:
This indentation error is causing the CI tests to fail. Can you kindly fix these?

Reply:
Hey @hmahmood24 Thanks for reviewing. I have solved the error. Let me know if there are any other changes required.